src/utils/webviewerUtils.ts

```diff
@@ -35,31 +35,40 @@ export async function blobToDocument(
   blob: Futurable<Blob>,
   extension: string,
   l?: string,
-): Promise<CoreControls.Document> {
-  const coreControls = window.CoreControls;
+): Promise<Core.Document | undefined> {
+  const coreControls = window.Core;
   const fetchedBlob = await blob;
-  return await coreControls.createDocument(fetchedBlob, { extension, l: l || globalLicense.get() });
+  try {
+    const document = await coreControls.createDocument(fetchedBlob, { extension, l: l || globalLicense.get() });
```

Comment:
this is the line throwing that uncaught exception:
```
File extension html is only supported by using WebViewer with WebViewer Server. See http://r.pdftron.com/fileformats for a full list of file formats supported by WebViewer. Visit https://www.pdftron.com/documentation/web/guides/wv-server-deployment for more information about WebViewer Server
```

changed the typing to be Core.Document | undefined to handle it